controllers/authorization.py

```diff
@@ -7,52 +9,140 @@
 
 
 class UpdateAuthorizationRequestBody(BaseModel):
+    """Request body model."""
     authorization: dict
     app_id: str
+    email: str
+
+
+@router.get('/authorizations', tags=['Authorization'])
+def get_accounts_with_authorization(response: Response,
+                                    app_id: str,
+                                    db_filter: str,
+                                    value: str,
+                                    authorization: str = Header(default=None)):
+    """Gets all accounts with specified authorizations.
```

Comment:
nit: I noticed some comments have a newline after the `"""`

Let's find a convention for it. I like the newlines 👀